specification/sbi_atee.adoc

```diff
@@ -143,13 +183,13 @@ shared memory regions are defined by the TVM using the EXT_TEE_GUEST extension.
 
 ===== Handling MMIO faults
 TVMs can define MMIO regions using the EXT_TEE_GUEST extension, and a runtime access to such a region causes
-a resumable exit from the TVM. The host can examine the exit code and `scause`, update the per-VCPU
-shared-memory region as appropriate, and resume TVM execution. This may involve instruction decoding
-using the information from the shared-memory region.
+a resumable exit from the TVM. The host can examine the exit code and `scause`, update the NACL
+shared memory region as appropriate, and resume TVM execution. This may involve instruction decoding
+using the information from the NACL shared memory region.
```

Comment:
Better to explicitly mentioned htinst read from NACL shared memory region. May be we can have a separate section to point out which registers are read/written by host & salus during which call for further clarificaiton.